CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+All notable changes to the Zowe Installer will be documented in this file.
+
+## Recent Changes
+
+- When `localhost` cannot be resolved use the IP address instead.
```

Comment:
I think it's better to say `hostname` than `localhost`.  The `localhost` is an alias for the loopback address 127.0.0.1, but `hostname` is a command that returns the name of the system on the network, e.g. S0W1.DAL-EBIS.IHOST.COM

Reply:
Tks for the feedback @John-A-Davies - I've made some updates so let me know if the latest push is any better.